.github/workflows/my_CI_pipelines.yml

```diff
@@ -71,4 +71,40 @@ jobs:
       
       - name: Get webhook url and run curl command
         run: |
-          curl -dH --fail -X POST '${{ secrets.WEBHOOK_URL }}'
+          curl -dH --fail -X POST "${{ secrets.TERRAFORMED_WEBHOOK_URL }}"
+
+  terraform_job:
+      name: Terraform Setup, Init, and Apply
+      runs-on: ubuntu-latest
+      needs: [build, security_check]  # Wait for previous jobs to complete
```

Comment:
I can see you've got this running alongside your `Publish Docker Image` stage - interesting to consider whether there is any reason we might prefer to have those run in series rather than parallel?